liteiclink/serwb/phy.py

```diff
@@ -366,6 +367,16 @@ def __init__(self, device, pads, mode="master", init_timeout=2**15, serdes_data_
         elif device[:3] == "xc7":
             taps = 32
             self.submodules.serdes = S7Serdes(pads, mode, serdes_data_width)
+        elif device[:4] == "xc6s":
+            # See Design Advisory for Spartan-6 Table2:
+            # https://support.xilinx.com/s/article/38408?language=en_US
+            max_taps = {188: 107, 200: 101, 266:72, 333: 54, 400: 43, 533: 28, 625: 22,
+                        667: 20, 800: 14, 945: 9, 1000: 8, 1050: 7, 1080: 6}
+            sys_freq = 50 # todo: how to get that information?
```

Comment:
Does somebody know a good way to get that information, or is it ok to make it a parameter?

Reply:
Hi @cklarhorst,

that's indeed a point I want to improve and avoid having to pass sys_clk_freq parameter to all module that require it. (ie having access to it globally when elaborating a design). I'll try to work on such improvements in the future, but for now for this module, if sys_clk_freq is required, you can add an argument (and make it optional with a default value of None).